packages/cli/api-importers/openapi/openapi-ir-to-fern-tests/src/__test__/__snapshots__/openapi/deepgram.json

```diff
@@ -10220,9 +10216,8 @@ channel:
                     "type": "optional<string>",
                   },
                   "keywords": {
-                    "allow-multiple": true,
```

Comment:
Discussed offline and caught this after the fact - we should _not_ be dropping the `allow-multiple` key in this case. We should continue to specify the type as `string`.